docs/FAQ/GitopsFAQ.md

```diff
@@ -88,9 +88,9 @@ Also, if you're using the optional _commit-status-tracker_ controller, please se
 
 https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets
 
-## ArgoCD is failing with forbidden: User "system:serviceaccount:openshift-gitops:argocd-application-controller" cannot create resource in API group
+## Argo CD is failing with forbidden: User "system:serviceaccount:openshift-gitops:argocd-application-controller" cannot create resource in API group
 
-The ArgoCD operator doesn't setup ArgoCD with enough permissions, this is a
+The Argo CD operator doesn't setup Argo CD with enough permissions, this is a
```

Comment:
@reginapizza @chetan-rns I don't think we need this workaround.

Reply:
Yeah. It's no longer required